src/CONST.ts

```diff
@@ -1665,6 +1665,9 @@ const CONST = {
             // Often referred to as "collect" workspaces
             TEAM: 'team',
         },
+        TAG: {
+            REQUIRED: 'required',
+        },
```

Comment:
```suggestion
```
Not used

Reply:
https://github.com/Expensify/App/pull/42972/commits/e2e37f255558f369708a5cb7cef3c91259a17b6f